compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -368,7 +368,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
             );
         };
         if let Some(file) = file {
-            err.note(format!("the full type name has been written to '{}'", file.display()));
+            err.note(format!("the full type name has been written to '{}' consider using --verbose to print it to the console", file.display()));
```

Comment:
actually this is two sentences in one. maybe make this a note? or at least it needs like a semicolon or something to separate it.

Reply:
I thought making it a generic note.. by using `DUMMY_SP` but then the context would not be clear. Probably adding it as an extra note below this one might be nice.

Reply:
Done!